### ISSUE TITLE
Typescript typings missing from 1.2.0 on yarnpkg.com

### ISSUE THREAD

**Issue by chrismay** [final-form/final-form-calculate#19]
### What is the current behavior?

the final-form-calculate v1.2.0 package, when installed via yarn or npm, does not include typescript typings.

### What is the expected behavior?

Typings should be included in 1.2.0 as per PR #12 

### Other information
```

$ curl -s https://registry.yarnpkg.com/final-form-calculate/-/final-form-calculate-1.2.0.tgz  | tar ztv
-rw-r--r-- 0/0            2467 1985-10-26 09:15 package/package.json
-rw-r--r-- 0/0            1071 1985-10-26 09:15 package/LICENSE
-rw-r--r-- 0/0            4719 1985-10-26 09:15 package/README.md
-rw-r--r-- 0/0            3794 1985-10-26 09:15 package/dist/final-form-calculate.cjs.js
-rw-r--r-- 0/0             192 1985-10-26 09:15 package/dist/final-form-calculate.cjs.js.flow
-rw-r--r-- 0/0            3712 1985-10-26 09:15 package/dist/final-form-calculate.es.js
-rw-r--r-- 0/0             192 1985-10-26 09:15 package/dist/final-form-calculate.es.js.flow
-rw-r--r-- 0/0            4342 1985-10-26 09:15 package/dist/final-form-calculate.umd.js
-rw-r--r-- 0/0             242 1985-10-26 09:15 package/dist/final-form-calculate.umd.js.map
-rw-r--r-- 0/0            1728 1985-10-26 09:15 package/dist/final-form-calculate.umd.min.js
-rw-r--r-- 0/0             112 1985-10-26 09:15 package/dist/final-form-calculate.umd.min.js.map
-rw-r--r-- 0/0             192 1985-10-26 09:15 package/dist/index.js.flow
-rw-r--r-- 0/0             448 1985-10-26 09:15 package/dist/types.js.flow
```

-  no `package/dist/index.d.ts`

Building locally seems to include the `dist/index.d.ts` file as expected. 

The package.json file says that it is `"version": "1.2.0"`, but other files (e.g. `dist/final-form-calculate.umd.js` seem different to what's in master at the moment - in fact they look very similar (identical?) to v1.1.0

**Comment by erikras** [comment]
That's pretty bizarre. The build script clearly copies it to the `dist` folder, and does so when I run it. `npm` weirdness, I guess...

**Comment by erikras** [comment]
Should be fixed in [`v1.2.1`](https://github.com/final-form/final-form-calculate/releases/tag/v1.2.1).

**Comment by chrismay** [comment]
Thanks for the swift fix - 1.2.1 looks fine :-)
